### DISCUSSION TITLE
Open Zoom Dev, with JITSI

### DISCUSSION THREAD

**Discussion by tidypy** [code4recovery/12-step-meeting-list#529] [Ideas]
I'm interested in developing Jitsi; an Open-Source 8x8 and Zoom Competitor.  

There are enhancements needed for Zoom concerning Integrations; 

(Concerning:  Data capture, payment, traditions, structure, administration, Voting Group Conscience, tradition 4, talkie 5 min timer, editing, streaming, closed caption, etc...)

I have a list of them.

Please reach out to me if interested in developing a 'Specific' platform for holding meetings online.  

Thank you kindly;
TidyPy

**Comment by tech2serve** [comment]
Hey @tidypy,

Thanks for the idea. This had been bounced around when online meetings started taking off, but we didn’t really have the bandwidth to do anything.

I’m reply to your email with more thoughts as any new project like this would not fall directly under TSML.
